src/sidebar/panel.js

```diff
@@ -111,12 +111,16 @@ ClassicEditor.create(document.querySelector('#editor'), {
             clearTimeout(loginTimeout);
             // set title attr of footer to the currently logged in account
             footerButtons.title = eventData.profile && eventData.profile.email;
+            localStorage.setItem('userEmail',footerButtons.title);
```

Comment:
nit: add a space after the comma

Reply:
This is a very stupid question but what is `nit`. And I'll surely make the changes

Reply:
nit stand for nit-pick, it is a non really import comment.
Instead that in this case the linter might fail.